web/viewer.html

```diff
@@ -317,6 +317,18 @@
                       </div>
                     </div>
                   </div>
+                  <div id="editorSignature" class="toolbarButtonWithContainer" hidden="true">
+                    <button id="editorSignatureButton" class="toolbarButton" type="button" disabled="disabled" title="Add or edit images" role="radio" aria-expanded="false" aria-haspopup="true" aria-controls="editorSignatureParamsToolbar" tabindex="0">
+                      <span>Add or edit signatures</span>
+                    </button>
+                    <div class="editorParamsToolbar hidden doorHangerRight menu" id="editorSignatureParamsToolbar">
+                      <div class="menuContainer">
+                        <button id="editorSignatureAddSignature" class="toolbarButton labeled" type="button" title="Add signature" tabindex="0">
+                          <span class="editorParamsLabel">Add signature</span>
```

Comment:
None of the new buttons/spans have l10n-attributes, nor entries in `en-US` Fluent file.

Reply:
Yep this part will change with the new UI.